x-pack/plugins/file_upload/public/components/import_complete_view.tsx

```diff
@@ -112,15 +164,45 @@ export class ImportCompleteView extends Component<Props, {}> {
         })
       : '';
 
-    return `${successMsg} ${failedFeaturesMsg}`;
+    return (
+      <EuiCallOut
+        title={i18n.translate('xpack.fileUpload.uploadSuccessTitle', {
+          defaultMessage: 'File upload complete',
+        })}
+      >
+        <p>{`${successMsg} ${failedFeaturesMsg}`}</p>
+      </EuiCallOut>
+    );
+  }
+
+  _renderIndexManagementMsg() {
+    return this.props.importResults && this.props.importResults.success ? (
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.fileUpload.jsonImport.indexModsMsg"
```

Comment:
Slightly different `i18n` ids in this file. This one and a couple others use `jsonImport`, others don't use it. Maybe switch them all to use `importComplete` or something similar?